Domain Connect Spec Draft.adoc

```diff
@@ -175,13 +178,12 @@ a domain name.
 
 === Case Sensitivity
 
-All values are case sensitive, with the following exceptions:
+All JSON object names, JSON object values, variable names, query parameters and query parameter values are case sensitive, with the following exceptions:
 
-* The following data passed in query strings:
-** The value of the serviceId, providerId, domain and host
-** All variable names
-* Variable names in templates are case insensitive
-* All variable names/keys in JSON documents are case insensitive
+* When retrieving the template from the DomainConnect Template Repository, the
+providerId and serviceId are all lower case. Regardless of how the template is
+retrieved, the providerId and serviceId are case sensitive.
+* Domain and host values for application of templates, as domains and sub-domains are not case sensitive

```

Comment:
I don't think this is true for 1&1.  1&1 stores the templates in a case sensitive manner.  

I would leave out the comment on the providerId and serviceId out of this section.